L/LLVM/LLVM_full@15/bundled/patches/0101-fix-darwin-tsan-on-old-sdks.patch

```diff
@@ -0,0 +1,66 @@
+From 36d2fc022bb0437333276db8c75fb43b338da343 Mon Sep 17 00:00:00 2001
+From: Julian Lettner <julian.lettner@apple.com>
+Date: Wed, 5 Feb 2020 08:17:06 -0800
+Subject: [PATCH 2/2] [TSan] Ensure we can compile the runtime with older SDKs
+
+One of my changes [1] included in this release silently bumped the
+minimal macOS SDK required for building the TSan runtime to SDK 10.12.
```

Comment:
And this one as well can be dropped now.

Reply:
This is probably needed because of https://github.com/JuliaPackaging/Yggdrasil/pull/5592#discussion_r1106242489